guides/hack/30-silencing-errors/01-introduction.md

```diff
@@ -120,6 +121,48 @@ strict mode checks in partial files by using the error code in
 error_codes_treated_strictly = 1002, 2045, 2055, 2060, 4005
 ```
 
+## Silencing Errors with `HH\FIXME\UNSAFE_CAST`
+
+```
+takes_int(HH\FIXME\UNSAFE_CAST<string,int>("foo",  "Your explanation here"));
+```
+
+To silence an error arising from a type mismatch on a particular expression, 
+add a call to `HH\FIXME\UNSAFE_CAST` with the expression as the first argument,
+an optional (string literal) comment, and explicit type hints indicating the 
+actual type of the expression and the expected type.
+
+The `UNSAFE_CAST` function also **has no runtime effect**. However, in contrast 
+to `HH_FIXME` comments, the `UNSAFE_CAST` function _does_ change the type of the 
+expression.
+
+### Silencing Errors per Expression
```

Comment:
Can you move this above the 'with comments' part?

While it's a less general approach, people will often stop reading when they find the first thing that works, so we should make it so they see this first.